source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/ObjectModelTest.cpp

```diff
@@ -1142,6 +1142,64 @@ namespace AdaptiveCardsSharedModelUnitTest
             }
         }
 
+        TEST_METHOD(ImplicitColumnTypeTest)
```

Comment:
Do we have tests that test column + bleed? Also do ensure the "corner" cases that we've identified in our various discussions are captured somewhere

Reply:
This particular case is a test for an old scenario. I touched the code for it in my refactoring so i added this test to make sure i didn't break it. The tests for column bleed are in the ContainerStyleTest.cpp file.

---
In reply to: [279799340](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r279799340) [](ancestors = 279799340)

Reply:
We also have the comprehensive bleed test file which tests the bleed scenarios for UWP.

---
In reply to: [280105918](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r280105918) [](ancestors = 280105918,279799340)